srtp/srtp.c

```diff
@@ -1006,6 +1006,10 @@ srtp_stream_init_keys(srtp_stream_ctx_t *srtp, srtp_master_key_t *master_key,
       rtp_xtn_hdr_base_key_len = base_key_length(session_keys->rtp_xtn_hdr_cipher->type,
                                                  rtp_xtn_hdr_keylen);
       rtp_xtn_hdr_salt_len = rtp_xtn_hdr_keylen - rtp_xtn_hdr_base_key_len;
+      if (rtp_xtn_hdr_salt_len > rtp_salt_len) {
```

Comment:
This would be correct if session_keys->rtp_cipher->type == SRTP_AES_GCM_128 or SRTP_AES_GCM_256, but might not be correct for another cipher.  Presently, we do not have this issue with other ciphers, but I still think it might be better to have an explicit check here so people in the future will understand why this comparison is done.